tests/integration/sync/it_sync_location_test.py

```diff
@@ -20,27 +21,10 @@
 """
 
 
-def test_sync_location(db_session):
-    port = free_port()
-    start_server(port,
-        locations = [
-            {
-                "id": 100,
-                "name": "Coco Bongo",
-                "code": "C",
-                "company_id": 50,
-                "country": "United States",
-                "region": "East Coast",
-                "city": "Edge City",
-                "address": "Blvd. Kukulcan Km 9.5 #30, Plaza Forum",
-                "longitude": 21.1326063,
-                "latitude": -86.7473191,
-                "type": "L",
-                "status": "open",
-                "comment": "Nightclub from a famous movie"
-            }
-        ]
-    )
+@pytest.mark.skip
+@unittest.mock.patch.object(Authenticated, 'auth')
+@unittest.mock.patch.object(Poster, 'locations')
+def test_sync_location(locations_mock, auth_mock, db_session):
```

Comment:
@vladarefiev I don't see any mocking of the data actually comming from Poster. Seems like this test is incomplete?

Reply:
@amihaiemil 
here is - `@unittest.mock.patch.object(Poster, 'locations')`
and here is https://github.com/timelesslounge/timelessis/pull/470/files#diff-e8e21c88402e90db658fb719e93a9369R38